src/azure-cli-testsdk/azure/cli/testsdk/recording_processors.py

```diff
@@ -1,156 +1 @@
-                "!!! The response body has been omitted from the recording because it is larger " \
-                "than {} KB. It will be replaced with blank content of {} bytes while replay. " \
-                "{}{}".format(self._max_response_body, length, self.control_flag, length)
-
-        return response
-
-
-class LargeResponseBodyReplacer(RecordingProcessor):
-    def process_response(self, response):
-        import six
-        body = response['body']['string']
-
-        # backward compatibility. under 2.7 response body is unicode, under 3.5 response body is
-        # bytes. when set the value back, the same type must be used.
-        body_is_string = isinstance(body, six.string_types)
-
-        content_in_string = (response['body']['string'] or b'').decode('utf-8')
-        index = content_in_string.find(LargeResponseBodyProcessor.control_flag)
-
-        if index > -1:
-            length = int(content_in_string[index + len(LargeResponseBodyProcessor.control_flag):])
-            if body_is_string:
-                response['body']['string'] = '0' * length
-            else:
-                response['body']['string'] = bytes([0] * length)
-
-        return response
-
-
-class OAuthRequestResponsesFilter(RecordingProcessor):
-    """Remove oauth authentication requests and responses from recording."""
-
-    def process_request(self, request):
-        # filter request like:
-        # GET https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/oauth2/token
-        import re
-        if not re.match('https://login.microsoftonline.com/([^/]+)/oauth2/token', request.uri):
-            return request
-
-
-class DeploymentNameReplacer(RecordingProcessor):
-    """Replace the random deployment name with a fixed mock name."""
-    def process_request(self, request):
-        import re
-        request.uri = re.sub('/deployments/([^/?]+)', '/deployments/mock-deployment', request.uri)
-        return request
-
-
-class GeneralNameReplacer(RecordingProcessor):
-    def __init__(self):
-        self.names_name = []
-
-    def register_name_pair(self, old, new):
-        self.names_name.append((old, new))
-
-    def process_request(self, request):
-        for old, new in self.names_name:
-            request.uri = request.uri.replace(old, new)
-
-            if request.body:
-                body = str(request.body)
-                if old in body:
-                    request.body = body.replace(old, new)
-
-        return request
-
-    def process_response(self, response):
-        for old, new in self.names_name:
-            if response['body']['string']:
-                response['body']['string'] = response['body']['string'].replace(old, new)
-
-            self.replace_header(response, 'location', old, new)
-            self.replace_header(response, 'azure-asyncoperation', old, new)
-
-        return response
+from azure_devtools.scenario_tests.recording_processors import *
```

Comment:
In the absence of someone firmly telling me otherwise I'm going to take these "import-through"s back out. Leaving them in was for the sake of caution but if @troydai wants them out we can probably assume it's safe.